internal/operands/template-validator/reconcile.go

```diff
@@ -137,6 +138,11 @@ func reconcileService(request *common.Request) (common.ResourceStatus, error) {
 func reconcileDeployment(request *common.Request) (common.ResourceStatus, error) {
 	validatorSpec := request.Instance.Spec.TemplateValidator
 	image := getTemplateValidatorImage()
+	request.Logger.Info(fmt.Sprintf("Template-Validator image: %s", image))
+	request.Logger.Info(fmt.Sprintf("VALIDATOR_IMAGE Envirnoment variable: %s", os.Getenv(common.TemplateValidatorImageKey)))
```

Comment:
These lines will pollute the log, because they will be written on each reconcilation iteration, but the image is not changing.

Can you move the logging somewhere where it will be called only once at the start of the operator?

Reply:
These were put here to make sure we know for this PR which image is being used, once this works fine, I think having them inside the panic block only is fine